model/api/src/main/java/org/jboss/aerogear/unifiedpush/api/PushMessageInformation.java

```diff
@@ -36,6 +36,10 @@
     private Date submitDate = new Date();
     private long totalReceivers;
 
+    private long appOpenCounter;
+    private Date firstOpenDate;
```

Comment:
`firstOpenDate` will never be updated once set , remember it's just a single instance regardless on how many people received it. The `latestOpen` will be updated each time.